Marketplace/Marketplace Updates/Marketplace Updates February 2017.md

```diff
@@ -6,7 +6,7 @@
 "contentIsHTML": false
 }}}
 
-The [CenturyLink Marketplace](/marketplace/) provides self-service, on-demand provisioning of provider services on the CenturyLink Platform. Below are the current products available in the Marketplace. The Marketplace is constantly evolving with more products, providers, and offerings. Check the release notes for updates. If you would like to join the CenturyLink Cloud Marketplace Provider Program, you can learn more [here](/marketplace-program/).
+The [CenturyLink Marketplace](/marketplace/) provides self-service, on-demand provisioning of provider services on the CenturyLink Platform. Below are the current products available in the Marketplace. The Marketplace is constantly evolving with more products, providers, and offerings. Check the Marketplace Updates for the latest Marketplace offerings. If you would like to join the CenturyLink Cloud Marketplace Provider Program, you can learn more [here](/marketplace-program/).
```

Comment:
Change `/marketplace/` to `https://www.ctl.io/marketplace/`
Change `/marketplace-program/` to `https://www.ctl.io/marketplace-program/`

Also on line 15 change `/marketplace/partner/ZZNC/product/CoIP/v/3.4.3/` to https://www.ctl.io/marketplace/partner/ZZNC/product/CoIP/v/3.4.3/`